webofneeds/won-bot/src/main/java/won/bot/framework/extensions/serviceatom/filter/ServiceAtomCreatedAtomRelationFilter.java

```diff
@@ -1,26 +1,22 @@
-package won.bot.framework.extensions.serviceatom;
+package won.bot.framework.extensions.serviceatom.filter;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import won.bot.framework.bot.context.BotContextWrapper;
 import won.bot.framework.eventbot.EventListenerContext;
 import won.bot.framework.eventbot.event.Event;
 import won.bot.framework.eventbot.event.MessageEvent;
 import won.bot.framework.eventbot.filter.impl.EventListenerContextAwareFilter;
+import won.bot.framework.extensions.serviceatom.ServiceAtomContext;
 import won.protocol.message.WonMessage;
 
-import java.lang.invoke.MethodHandles;
 import java.net.URI;
 import java.util.Objects;
 
 /**
  * Filter that only accepts Events between the serviceAtom and other
  * createdAtoms
  */
-public class ServiceAtomRelatedFilter extends EventListenerContextAwareFilter {
-    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
-
-    ServiceAtomRelatedFilter(EventListenerContext context) {
+public class ServiceAtomCreatedAtomRelationFilter extends EventListenerContextAwareFilter {
```

Comment:
Class name is long and still unclear - does this filter for a relationship between a serviceAtom and atoms created by the bot? Or for a relationship between serviceAtom and user created atoms? 

The method does not seem to check whether the second involved atom is a specific kind of atom, only if it is known to the bot, but it checks whether the Event sent is a MessageEvent. Alternative name suggestions for this class, depending on it's use: `ServiceAtomMessageFilter`, `ServiceUserMessageFilter`, ServiceAtomControlMessageFilter`